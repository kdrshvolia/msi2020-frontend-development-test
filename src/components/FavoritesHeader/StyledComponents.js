import styled, { css } from 'styled-components';const FavoritedHeader = styled.h3`  width: 84px;  font-weight: 500;  font-size: 20px;  line-height: 28px;  color: #ababab;  margin: 0;`;export { FavoritedHeader };