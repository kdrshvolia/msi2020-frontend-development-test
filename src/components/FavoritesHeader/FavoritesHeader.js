import React from 'react';import PropTypes from 'prop-types';import { FavoritedHeader } from './StyledComponents';const FavoritesHeader = ({ children }) => {  return <FavoritedHeader>{children}</FavoritedHeader>;};FavoritedHeader.propTypes = {  children: PropTypes.string.isRequired,};export default FavoritesHeader;