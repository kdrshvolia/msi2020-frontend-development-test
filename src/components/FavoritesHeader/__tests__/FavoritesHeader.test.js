import React from 'react';import renderer from 'react-test-renderer';import FavoritesHeader from '../FavoritesHeader';describe('FavoritesHeader', () => {  it('Renders correctly', () => {    const rendered = renderer.create(<FavoritesHeader>Some text</FavoritesHeader>);    expect(rendered).toMatchSnapshot();  });});