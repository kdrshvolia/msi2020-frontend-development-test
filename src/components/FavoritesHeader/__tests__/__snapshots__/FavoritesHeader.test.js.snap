// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesHeader Renders correctly 1`] = `
<h3
  className="sc-AxjAm fBabhS"
>
  Some text
</h3>
`;
